src/gen_esme.erl

```diff
@@ -173,6 +173,7 @@ opts_to_state(Mod, Opts) ->
     BindTimeout = maps:get(bind_timeout, Opts, ?BIND_TIMEOUT),
     ReqPerSec = maps:get(req_per_sec, Opts, undefined),
     MaxAwaitReqs = maps:get(max_await_reqs, Opts, undefined),
+    TimeoutTolerance = maps:get(timeout_tolerance, Opts, false),
```

Comment:
The name seems a bit abstract. Lets rename it to something like `ignore_response_timeout` since we specifically ignore `response_timer` from smpp spec (table in 7.2 from v3.4).